pkg/core/blockchain_test.go

```diff
@@ -337,6 +338,23 @@ func TestVerifyTx(t *testing.T) {
 		tx.Scripts[0].InvocationScript[10] = ^tx.Scripts[0].InvocationScript[10]
 		checkErr(t, ErrVerificationFailed, tx)
 	})
+	t.Run("OldTX", func(t *testing.T) {
```

Comment:
`ErrOOM` is also easy to cover.

Reply:
Requires a little hack, but yeah, makes sense.